src/CLICommand/cli_command.ts

```diff
@@ -25,8 +25,23 @@ program.addHelpCommand(false);
  */
 function setCommanderCommand(commandDescriptor: CommandDescriptor, program: CliApiObject): void {
 	let command: CliApiObject = program.command(commandDescriptor.name);
-	commandDescriptor.parameters.forEach((parameterName) => {
-		const parameter = new Parameter(parameterName);
+	commandDescriptor.parameters.forEach((p) => {
```

Comment:
Please use a more descriptive name than 'p'. I'm surprised the linter allowed it.